### PR TITLE
Upgrading dma lvc

### PR DESCRIPTION
**Pull request by SLC-Laurens-Vercruysse**
This pull request contains the updated upgrade paths and best practices page, as well as an updated VerifyNatsIsRunning prerequisite page. 
Note however, that the prerequisites for which Core Eco is not accountable have not yet been filled in. 
Likely we'll have to request feedback from the domains in question.

### CODE REVIEW

File: user-guide/Advanced_Functionality/DataMiner_Agents/Upgrading_a_DMA/Preparing_to_upgrade_a_DataMiner_Agent.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+> [!TIP]
+> For more information on how to downgrade your DMS using a VM snapshot restore, see [Downgrade with VM snapshot restore](xref:MOP_Downgrading_a_DMS#downgrade-with-vm-snapshot-restore-preferred).
+
+> [!NOTE]
+> The timing for a VM snapshot restore depends on your specific setup.
+
+### DataMiner backup
+
+To make a backup of a DataMiner Agent using DataMiner Taskbar Utility or DataMiner Cube, see [Backing up a DataMiner Agent](xref:Backing_up_a_DataMiner_Agent).
```
Comment (EdithLansens) [inline user-guide/Advanced_Functionality/DataMiner_Agents/Upgrading_a_DMA/Preparing_to_upgrade_a_DataMiner_Agent.md:67]: For the sake of continuity, and if this is useful here, we could perhaps add a tip with more information on how to downgrade a DMS using a DataMiner backup as well? I am however unsure of whether to add a link to [Restoring a DataMiner Agent](https://docs.dataminer.services/user-guide/Advanced_Functionality/DataMiner_Agents/Restoring_a_DataMiner_Agent.html) or to [Downgrading a DMS](https://docs.dataminer.services/user-guide/Reference/MOPs/MOP_Downgrading_a_DMS.html).

File: user-guide/Advanced_Functionality/DataMiner_Agents/Upgrading_a_DMA/Upgrade_Paths/Upgrade_path_960-96x_to_1030-103x.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+Make sure the IP network ports 9090, 4222, 6222, and 8222 (NATS monitoring only) are opened, as explained in [Configuring the IP network ports](xref:Configuring_the_IP_network_ports).
+
+> [!TIP]
+> See also: [Checking the required open ports in a DMS](xref:MOP_Checking_the_required_open_ports_in_a_DMS)
+
+## Upgrade path
+
+Never directly upgrade to a major DataMiner version that is several versions higher than the current version. To prevent issues and make it easier to find the root cause if anything goes wrong after an upgrade, we highly recommend that you follow this upgrade path:
+
+DataMiner 9.6.0/9.6.x -> DataMiner 10.0.0 [CU19] -> DataMiner 10.2.0/10.2.x
```
Comment (EdithLansens) [inline user-guide/Advanced_Functionality/DataMiner_Agents/Upgrading_a_DMA/Upgrade_Paths/Upgrade_path_960-96x_to_1030-103x.md:32]: This upgrade path is identical to the one for DataMiner 9.6.0/9.6.x to DataMiner 10.2.0/10.2.x. 
Is it as simple as changing the last part?

```suggestion
DataMiner 9.6.0/9.6.x -> DataMiner 10.0.0 [CU19] -> DataMiner 10.3.0/10.3.x
```

File: user-guide/Advanced_Functionality/DataMiner_Agents/Upgrading_a_DMA/Upgrade_Paths/Upgrade_path_960-96x_to_1020-102x.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+## NATS ports
+
+Make sure the IP network ports 9090, 4222, 6222, and 8222 (NATS monitoring only) are opened, as explained in [Configuring the IP network ports](xref:Configuring_the_IP_network_ports).
+
+> [!TIP]
+> See also: [Checking the required open ports in a DMS](xref:MOP_Checking_the_required_open_ports_in_a_DMS)
+
+## Upgrade path
+
+Never directly upgrade to a major DataMiner version that is several versions higher than the current version. To prevent issues and make it easier to find the root cause if anything goes wrong after an upgrade, we highly recommend that you follow this upgrade path:
```
Comment (EdithLansens) [inline user-guide/Advanced_Functionality/DataMiner_Agents/Upgrading_a_DMA/Upgrade_Paths/Upgrade_path_960-96x_to_1020-102x.md:26]: Is this still correct, seeing as you removed a similar sentence on the "Upgrading a DataMiner Agent" page?

File: user-guide/Advanced_Functionality/DataMiner_Agents/Upgrading_a_DMA/Upgrade_Paths/Upgrade_path_960-96x_to_1030-103x.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+## NATS ports
+
+Make sure the IP network ports 9090, 4222, 6222, and 8222 (NATS monitoring only) are opened, as explained in [Configuring the IP network ports](xref:Configuring_the_IP_network_ports).
+
+> [!TIP]
+> See also: [Checking the required open ports in a DMS](xref:MOP_Checking_the_required_open_ports_in_a_DMS)
+
+## Upgrade path
+
+Never directly upgrade to a major DataMiner version that is several versions higher than the current version. To prevent issues and make it easier to find the root cause if anything goes wrong after an upgrade, we highly recommend that you follow this upgrade path:
```
Comment (EdithLansens) [inline user-guide/Advanced_Functionality/DataMiner_Agents/Upgrading_a_DMA/Upgrade_Paths/Upgrade_path_960-96x_to_1030-103x.md:30]: Same question: Change needed?
```suggestion
Follow this upgrade path:
```

File: user-guide/Advanced_Functionality/DataMiner_Agents/Upgrading_a_DMA/Preparing_to_upgrade_a_DataMiner_Agent.md (hunk 1 above)
Comment (EdithLansens) [reply user-guide/Advanced_Functionality/DataMiner_Agents/Upgrading_a_DMA/Preparing_to_upgrade_a_DataMiner_Agent.md]: I've deleted all mentions to downgrading a DMS, as this was a misinterpretation on my part that could indeed be confusing and a no-go in the case of a DataMiner backup. See [commit](https://github.com/SkylineCommunications/dataminer-docs/pull/1188/commits/a339ad7ec18dde526354d2361490d41cfa2b0cc1).

**Review by robin-devos-skyline — CHANGES_REQUESTED** [review]
I updated this section based on DCP207984 
Also, there's a (hidden) comment in there with context + RN reference, not sure if it'll go noticed :)

File: user-guide/Advanced_Functionality/DataMiner_Agents/Upgrading_a_DMA/Upgrade_Paths/Upgrade_path_960-96x_to_1030-103x.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+Install the following packages:
+
+- [Microsoft .NET 4.8](https://go.microsoft.com/fwlink/?linkid=2088631)
+
+- Install ASP.NET Core
+
+  - [ASP.NET Core 5.0.11](https://download.visualstudio.microsoft.com/download/pr/df452763-4b7d-490a-bc03-bd1003d3ff4c/665ee1786528809f33e791558b69cf51/dotnet-hosting-5.0.11-win.exe): If you want to upgrade your DMA to a DataMiner version **prior to DataMiner 10.3.3**
+
+  - [ASP.NET Core 6.0.13](https://download.visualstudio.microsoft.com/download/pr/0cb3c095-c4f4-4d55-929b-3b4888a7b5f1/4156664d6bfcb46b63916a8cd43f8305/dotnet-hosting-6.0.13-win.exe): If you want to upgrade your DMA to **DataMiner 10.3.3 or higher**
```
[the remark this answers is not in the document]
Comment (SLC-Laurens-Vercruysse) [reply user-guide/Advanced_Functionality/DataMiner_Agents/Upgrading_a_DMA/Upgrade_Paths/Upgrade_path_960-96x_to_1030-103x.md:16]: Correct suggestion. Should be updated as such.

**Review by SLC-Laurens-Vercruysse — COMMENTED** [review]
Changes looked good. Added comments below.

File: user-guide/Advanced_Functionality/DataMiner_Agents/Upgrading_a_DMA/Preparing_to_upgrade_a_DataMiner_Agent.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+As of now, the following prerequisites are available:
+
+- Verify Cluster Ports: Verifies whether the ports used by DataMiner can be reached in between DataMiner Agents. If this check fails, you will need to install the [VerifyClusterPorts.dmupgrade](xref:VerifyClusterPortsdmupgrade) package. From DataMiner 10.2.0 [CU2]/10.2.5 onwards, this prerequisite is available by default and runs automatically when you upgrade.
+
+- [Verify NATS is Running](xref:VerifyNatsIsRunning): Verifies whether the crucial NATS service used by DataMiner is running on all required DataMiner Agents. From DataMiner 10.2.0 [CU14]/10.2.7 onwards, this prerequisite is available by default and runs automatically when you upgrade.
+
+- [Verify Cloud DxM Version](xref:BPA_Verify_Cloud_DxM_Version): Verifies whether the minimum required version is installed for all DxMs in the system. From DataMiner 10.2.0 [CU6]/10.2.8 onwards, this prerequisite is available by default and runs automatically when you upgrade.
+
+- Service Automatic Properties
```
[the remark this answers is not in the document]
Comment (EdithLansens) [reply user-guide/Advanced_Functionality/DataMiner_Agents/Upgrading_a_DMA/Preparing_to_upgrade_a_DataMiner_Agent.md:45]: I will contact Seppe to add a similar page dedicated to this prerequisite, ideally in the foreseeable future. Could you provide me with a short description of the prerequisite that I can put here in the meantime?

**Pull request merged**